specification/compute/resource-manager/readme.az.md

```diff
@@ -2,14 +2,60 @@
 
 These settings apply only when `--az` is specified on the command line.
 
-``` yaml $(az) && $(target-mode) == "core"
+``` yaml $(az) && $(cloudservice)
+tag: package-2020-10-01-preview-only
+az:
+  extensions: cloud-service
+  namespace: azure.mgmt.compute
+  package-name: azure-mgmt-compute
+az-output-folder: $(azure-cli-extension-folder)/src/cloudservice
+python-sdk-output-folder: $(az-output-folder)/azext_cloudservice/vendored_sdks/cloudservice
+cli:
+    cli-directive:
+        - where:
+            group: "*"
+            op: "*"
+          hidden: true
+        - where:
+            group: "CloudServiceRoleInstances"
+            op: "*"
+          hidden: false
+        - where:
+            group: "CloudServiceRoles"
+            op: "*"
+          hidden: false
+        - where:
+            group: "CloudServices"
+            op: "*"
+          hidden: false
+        - where:
+            group: "CloudServicesUpdateDomain"
+            op: "*"
+          hidden: false
+directive:
+  - where:
+      group: cloud-service cloud-service-role-instance
+    set:
+      group: cloud-service role-instance
+  - where:
+      group: cloud-service cloud-service-role
+    set:
+      group: cloud-service role
+  - where:
+      group: cloud-service cloud-service-update-domain
+    set:
+      group: cloud-service update-domain
+```
+
+``` yaml $(az) && $(target-mode) == "core" && $(sshkey)
```

Comment:
may no need to add "&& $(sshkey)" here, all target-model==core resources shall share this section

Reply:
Can I delete $(target-mode) == "core"? I use $(sshkey) and $(cloudservice) to allow only generating one resource.

Reply:
$(target-mode) == "core" is also used by depth coverage pipeline.   
sshKey is not a single RP like cloudservice, so don't need to write a special section for sshKey

Reply:
Got it. I can't generate one command group independently. I need to regenerate all generated code.